pyfixdoc.py

```diff
@@ -468,7 +470,7 @@ def format_class (
                 line_num, obj_md = self.document_method(path_list, member_name, member_obj, func_kind)
                 obj_md_pos[line_num] = obj_md
 
-        for pos, obj_md in sorted(obj_md_pos.items()):
+        for _, obj_md in sorted(obj_md_pos.items()):
```

Comment:
see my comment on `_line_num`
```suggestion
        for _pos, obj_md in sorted(obj_md_pos.items()):
```

Reply:
Idem.